### DISCUSSION TITLE
Proposal: Making LottieAnimation for splash screen and app's icon

### DISCUSSION THREAD

**Discussion by AbdAlghaniAlbiek** [microsoft/microsoft-ui-xaml#7441] [Q&A]
Is it possible to add lottie animation to UWP and Win32 apps using WinUI in splash screen and in the app's icon like in Windows 11 ?

**Comment by StephenLPeters** [comment]
It is definitely possible, the LottieViewer application from the Microsoft Store does it.  @nishitha-burman do you know how this is done?

**Comment by nishitha-burman** [comment]
Currently there is no formal support for Lottie animations in splash screen. For Lottie Viewer the animation is played once the app is launched.

**Comment by AbdAlghaniAlbiek** [comment]
@nishitha-burman So.. do you have plans to add these (Lottie animations in app's icon and splash screen) as features in the future?

**Comment by harvinders** [comment]
It would be great to see these as features in WinUI, or at least there should be an easy way for developers to add these in their application

**Comment by nishitha-burman** [comment]
Currently there is no immediate plan to support these but they are in the backlog of features we are looking into. 
@MarissaMatt for Lottie animations in app's icon and @samanthamsong for Lottie animations in splash screen

**Comment by AbdAlghaniAlbiek** [comment]
@nishitha-burman 
 thank you for your interest
